src/Makefile.am

```diff
@@ -2324,6 +2323,7 @@ tests_testHttpRequest_LDADD = \
 	format/libformat.la \
 	store/libstore.la \
 	sbuf/libsbuf.la \
+	time/libtime.la \
```

Comment:
I assume we need to link `time/libtime.la` because of `Ip::ProbeTransport()` debug output - but if not please try to avoid this extra change.

Reply:
> I assume we need to link time/libtime.la because of Ip::ProbeTransport() debug output

Your assumption is correct. See https://github.com/squid-cache/squid/commit/063b4c38fcbdd9a6e5e38025723b37e8877d63b2.